src/command_modules/azure-cli-monitor/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.0.15
+++++++
+* BREAKING CHANGE: Add multi-diagnostic settings support.
```

Comment:
Typically this gets abbreviated `BC`.  What is the breaking change?

Reply:
Additional `--name` in `monitor diagnostic-settings create` command now that multiple diagnostic settings are supported.